deploy-manage/maintenance/start-stop-services/start-stop-elasticsearch.md

```diff
@@ -6,23 +6,393 @@ mapped_urls:
 
-% GitHub issue: https://github.com/elastic/docs-projects/issues/353
+## Starting Elasticsearch [starting-elasticsearch]
 
-% Scope notes: Combine into one topic
+The method for starting {{es}} varies depending on how you installed it.
 
-% Use migrated content from existing pages that map to this page:
+### Archive packages (`.tar.gz`) [start-targz]
 
-% - [ ] ./raw-migrated-files/elasticsearch/elasticsearch-reference/starting-elasticsearch.md
-% - [ ] ./raw-migrated-files/elasticsearch/elasticsearch-reference/stopping-elasticsearch.md
+If you installed {{es}} with a `.tar.gz` package, you can start {{es}} from the command line.
 
-% Internal links rely on the following IDs being on this page (e.g. as a heading ID, paragraph ID, etc):
+#### Run {{es}} from the command line [_run_es_from_the_command_line]
 
-$$$start-deb$$$
+Run the following command to start {{es}} from the command line:
 
-$$$start-rpm$$$
+```sh
+./bin/elasticsearch
+```
 
-$$$_enroll_nodes_in_an_existing_cluster_3$$$
+When starting {{es}} for the first time, security features are enabled and configured by default. The following security configuration occurs automatically:
 
-$$$start-es-deb-systemd$$$
\ No newline at end of file
+* Authentication and authorization are enabled, and a password is generated for the `elastic` built-in superuser.
+* Certificates and keys for TLS are generated for the transport and HTTP layer, and TLS is enabled and configured with these keys and certificates.
+* An enrollment token is generated for {{kib}}, which is valid for 30 minutes.
+
+The password for the `elastic` user and the enrollment token for {{kib}} are output to your terminal.
+
+We recommend storing the `elastic` password as an environment variable in your shell. Example:
+
+```sh
+export ELASTIC_PASSWORD="your_password"
+```
+
+If you have password-protected the {{es}} keystore, you will be prompted to enter the keystore’s password. See [Secure settings](../../../deploy-manage/security/secure-settings.md) for more details.
+
+By default {{es}} prints its logs to the console (`stdout`) and to the `<cluster name>.log` file within the [logs directory](../../../deploy-manage/deploy/self-managed/important-settings-configuration.md#path-settings). {{es}} logs some information while it is starting, but after it has finished initializing it will continue to run in the foreground and won’t log anything further until something happens that is worth recording. While {{es}} is running you can interact with it through its HTTP interface which is on port `9200` by default.
+
+To stop {{es}}, press `Ctrl-C`.
+
+::::{note}
+All scripts packaged with {{es}} require a version of Bash that supports arrays and assume that Bash is available at `/bin/bash`. As such, Bash should be available at this path either directly or via a symbolic link.
+::::
+
+#### Enroll nodes in an existing cluster [_enroll_nodes_in_an_existing_cluster_3]
```

Comment:
we can remove this because it's the same content as the other page `/deploy-manage/maintenance/add-and-remove-elasticsearch-nodes.md`

in the original files, it used a common snippet that we don't need